VirtoCommerce.Platform.Core/Modularity/ModuleDescriptor.cs

```diff
@@ -65,7 +65,7 @@ public ModuleIdentity Identity
         public ICollection<string> ValidationErrors { get; set; }
         public bool IsRemovable { get; set; }
         public bool IsInstalled { get; set; }
-        public ModuleIdentity InstalledVersion { get; set; }
+        //public ModuleIdentity InstalledVersion { get; set; }
```

Comment:
Are you sure that this property not used anywhere?

Reply:
fix it, add NullIgnore to jsonsettings in BackgroundJob